odxtools/admindata.py

```diff
@@ -99,15 +100,16 @@ class CompanyRevisionInfo:
 
     @property
     def company_data(self) -> CompanyData:
-        assert self._company_data is not None
+        if self._company_data is None:
+            odxraise()
         return self._company_data
```

Comment:
Company data is a required element in XML?

Reply:
within COMPANY-REVISON-INFO, yes: COMPANY-DATA-REF is mandatory and if the reference is not resolved, the object has not been properly initialized. the check should IMO be removed, though: we will get an `AttributeException` if something went wrong, but on most occasions where the same pattern applies we do not check for this either. (that said, this would categorize as drive-by fix. ;))